05_syntax_lab/02.py

```diff
@@ -1,6 +1,15 @@
-"""
-Write a Python program that randomizes 7 numbers
-and prints their sum total.
-If the sum is divisable by 7, also print the word "Boom"
-"""
-
+"""
+Exercise 02
+
+- Random number between 1-100 + Boom if /7 = 0
+"""
+
+from random import randint
+
+sum = 0
+
+for i in range (0,7):
+	i = randint(1, 100)
+	print i
+if i %7 == 0:
+	print 'Boom'

```

Comment:
You are using the ```i``` variable for both the iterator and the generated random number. It is confusing.
Oh and you could write ```range(7)```
Line 14 is not checking ```sum```